src/mito2/src/cache/file_cache.rs

```diff
@@ -376,12 +439,18 @@ mod tests {
     fn test_cache_file_path() {
         let file_id = FileId::parse_str("3368731b-a556-42b8-a5df-9c31ce155095").unwrap();
         assert_eq!(
-            "test_dir/5299989643269.3368731b-a556-42b8-a5df-9c31ce155095",
-            cache_file_path("test_dir", (RegionId::new(1234, 5), file_id))
+            "test_dir/5299989643269.3368731b-a556-42b8-a5df-9c31ce155095.parquet",
+            cache_file_path(
+                "test_dir",
+                IndexKey::new(RegionId::new(1234, 5), file_id, FileType::Parquet)
+            )
         );
         assert_eq!(
-            "test_dir/5299989643269.3368731b-a556-42b8-a5df-9c31ce155095",
-            cache_file_path("test_dir/", (RegionId::new(1234, 5), file_id))
+            "test_dir/5299989643269.3368731b-a556-42b8-a5df-9c31ce155095.parquet",
```

Comment:
nitpick: Also test `5299989643269.3368731b-a556-42b8-a5df-9c31ce155095.puffin` and invalid suffix e.g. `5299989643269.3368731b-a556-42b8-a5df-9c31ce155095.parq`